pwndbg/heap/ptmalloc.py

```diff
@@ -658,17 +658,226 @@ def __init__(self) -> None:
         # ptmalloc cache for current thread
         self._thread_cache: gdb.Value = None
 
-    def largebin_size_range_from_index(self, index):
-        index += NSMALLBINS
-        spaces_table = self._spaces_table()
-        largest_largebin = self.largebin_index(pwndbg.gdblib.arch.ptrmask)
-        start_size = (NSMALLBINS * self.malloc_alignment) - self.malloc_alignment
+        """
+        Largebin reverse lookup tables.
+        These help determine the range of chunk sizes that each largebin can hold.
+        They were generated by running every chunk size between the minimum & maximum large chunk
+        sizes through largebin_index().
+        Largebin 31 (bin 95) isn't used on i386 when MALLOC_ALIGNMENT is 16, so its value must be added manually.
+        """
+        self.largebin_reverse_lookup_32 = (
```

Comment:
Note: perhaps we want to set it as class field so the tuple isn't recreated on each call to this function.